PerimeterXModule/PxModuleConfigurationSection.cs

```diff
@@ -53,6 +54,7 @@ public string AppId
 			set
 			{
 				this["appId"] = value;
+				this["collectorUrl"] = string.Format(CollectorUrl, value);
```

Comment:
Isn't this creating a potential override of the `this["collectorUrl"]` (and as such the `CollectorUrl`) value (if both "appId" and "collectorUrl" params are being set from the webconfig)?
Also, I think you can do this assign here only if you can make sure that `this["collectorUrl"]` is currently the DefaultValue (otherwise perhaps the `collectorUrl` received from the webconfig does not include a placeholder for the appId).

Reply:
Right, i changed the the attribute so it wont have a default value and initiate that value here
good catch